Common/tests/ArmoniK.Core.Common.Tests.csproj

```diff
@@ -13,9 +13,9 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="EphemeralMongo5" Version="0.1.3" />
```

Comment:
Is there a reason not to pick EphemeralMongo6?

Reply:
I believe the infra still uses mongo 5, so better keep it in sync ?

Reply:
Yes, the infra uses mongo 5